lua/md-pdf/init.lua

```diff
@@ -12,6 +12,58 @@ local viewer_open = false
 local conv_started = false
 local pdf_output_path = ""
 
+local function trim_quotes(str)
+    if not str then
+        return nil
+    end
+    str = vim.trim(str)
+    local first = str:sub(1, 1)
+    local last = str:sub(-1)
+    if (first == '"' and last == '"') or (first == "'" and last == "'") then
+        return str:sub(2, -2)
+    end
+    return str
+end
+
+local function resolve_logo_path(fullname, file_dir)
```

Comment:
Similarly here, please add type annotations.
This functions is neat, it does something header specific though, no?
How about naming it that way to reflect it! e.g., `resolve_header_logo(_path)`